themes/cdap/layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,31 @@
+{{/*
+  Copyright © 2015-2019 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+*/}}
+<!--
```

Comment:
yeah, this second style is correct, can we standardize on this? Remove the duplicate one above in this case.